features/up.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Setup project tools using Tuist
+
+  Scenario: The project is an iOS application with a custom test tool
```

Comment:
I'd remove the indentation from here.

Reply:
Ignore my comment then, for some reason I thought the indentation is not required there.